### DISCUSSION TITLE
Inconsistency Between Motif Zscore Distribution and Plotted on UMAP Embedding?

### DISCUSSION THREAD

**Discussion by mfrenkel16** [GreenleafLab/ArchR#1849] [Questions / Documentation]
I think I'm running into a strange inconsistency between a motif's Zscore when the distribution is plotted versus when it's plotted on top of the UMAP embedding. In this case, I know that the ATF1_118 motif is enriched in differentially accessible peaks in the group I'm calling "group A" (left-hand side graphs). You can see this in the distribution (z scores are consistently much higher in group A) and in the list of motifs (ATF1_118 is #6 on that list with p ~ 1e-200). Also (not shown) but in Loupe browser these group A cells have an enrichment of the ATF1 motif with p = 6.37e-17. However, on the right hand side is see something peculiar: 1. the footprinting at the ATF1_118 motif looks identical across all groups) and 2. when I plot the ATF_118 zscore on the UMAP embedding, it actually looks depleted in the group A cells (highlighted in the red dash) among other groups (that I know it doesn't shared similar Z scores with based on the distributions on the left hand side of the pannel). 

Has anyone encountered this inconsistency? Maybe it's hard to diagnose without my code, but I'm following the ArchR manual "Motif Deviation" section as closely as possible.

Also, I can't find what these motifs are. I know they're from "cisbp" but the cisbp website doesn't seem to share the same naming convention with ArchR (e.g. I see a few ATF1 motifs but nothing named ATF1_118; On cisbp's website I see ATF1 M08792_2.00 and ATF1 M09485_2.00 for instance).

Thanks!
[Weird_Results.pdf](https://github.com/GreenleafLab/ArchR/files/10778181/Weird_Results.pdf)

**Comment by rcorces** [comment]
You are looking at two different things. One is the enrichment of a motif in **differentially accessible peaks**, the other is the motif deviation across **all peaks**. I generally wouldnt default to trusting anything from Loupe browser. Motifs come from the `chromVARmotifs` package (https://github.com/GreenleafLab/chromVARmotifs).

**Reply by mfrenkel16** [reply]
Thanks for the quick reply! I get that for the bottom left graph, but I don't think that's the case for the top left, no? The top left and the  far right graphs are both plotting motif deviation across all peaks I think. In both cases, they are colored by the same "MotifMatrix" (which was created with `addDeviationsMatrix`). Unless I'm misunderstanding something, those two plots are the exact same data but just in one case showing it as a distribution and in the other plotting on top of the embedding. It's those two that seem inconsistent, but again I could be misunderstanding.

**Reply by rcorces** [reply]
Without a reproducible example, there isnt much I can help with here
